metadata/sandbox_test.go

```diff
@@ -120,7 +123,8 @@ func TestSandboxUpdate(t *testing.T) {
 			"ext1": &types.Any{TypeUrl: "url1", Value: []byte{1, 2}},
 			"ext2": &types.Any{TypeUrl: "url2", Value: []byte{4, 5, 6}},
 		},
-		Runtime: api.RuntimeOpts{Name: "test"},
+		Runtime:    api.RuntimeOpts{Name: "test"},
+		Controller: "remote",
```

Comment:
Sandboxes are implemented by shims (side by side with task service), so we can reuse `Runtime` field to find right shim for tasks/sandboxing. There is no need in dedicated `Controller` field.

Reply:
Reasonable. Thanks!